Add generic web site app, basic GitHub repo app

gregelin opened the pull request:
Created a generic web site app based on the Django web site app.

Created a basic GitHub repo app and added it to Django web site app and generic web site app.

gregelin committed:
Add generic-web-site app stub

gregelin committed:
Merge branch 'published' of github.com:GovReady/govready-apps-dev into ge_primary

gregelin committed:
Merge branch 'published' into ge_primary

gregelin committed:
Created GitHub Repo app

gregelin committed:
--amend

gregelin committed:
--amend

gregelin committed:
Create and polish basic GitHub repo app

gregelin committed:
Merge branch 'ge_github_app' of github.com:GovReady/govready-apps-dev into ge_github_app

gregelin committed:
Create and polish basic GitHub repo app

gregelin committed:
Create basic GitHub repo app, Merge into ge_primary

gregelin committed:
Add GitHub repo to generic-web-site

gregelin committed:
Add GitHub repo to generic-web-site

gregelin committed:
Merge branch 'ge_primary' of github.com:GovReady/govready-apps-dev into ge_primary

JoshData commented:
Merge is blocked here because a rebase-type merge would generate conflicts possible. Looks like you disabled a regular merge and a squash merge on this repo. Was that intentional? I'm not sure if GitHub will let us do those either - it might say there's a conflict anyway.

gregelin committed:
Improve generic and django website compliance apps

Add POAMS for 800-53
Add system maintenance module-set app
Add in draft ssp.yaml from wordpress website example
Move ssp placement under action-buttons
Add people and policy questions to generic-website and
django-website

gregelin commented:
Dropping this pull request and redoing separately.
